pkg/kn/commands/service/service_list_test.go

```diff
@@ -73,12 +76,13 @@ func TestServiceListDefaultOutput(t *testing.T) {
 	} else if !action.Matches("list", "services") {
 		t.Errorf("Bad action %v", action)
 	}
-	assert.Check(t, util.ContainsAll(output[0], "NAME", "DOMAIN", "GENERATION", "AGE", "CONDITIONS", "READY", "REASON"))
+	assert.Check(t, util.ContainsAll(output[0], "NAME", "ADDRESS", "GENERATION", "AGE", "CONDITIONS", "READY", "REASON"))
```

Comment:
How about using `URL` for column header ?
We are referring this field as `URL` in `kn route list`.
WDYT ?

Reply:
URL (`status.url`) and ADDRESS (`status.addess`) are different fields:

* `URL` is the externally routable endpoint for this service going over the ingress (http://hello.default.example.com)
* `ADDRESS` is the internal k8s service address (http://hello.default.svc.cluster.local)

In the list I think we should only show the externally routed URL, or ? So you are right, should be URL then (but have to check whether we are really picking up the right field)

Reply:
Changed to URL, PTAL.